src/main/java/org/gridsuite/study/server/networkmodificationtree/ModelNodeInfoRepositoryProxy.java

```diff
@@ -72,6 +75,11 @@ public void updateLoadFlowResultAndStatus(AbstractNode node, LoadFlowResult load
         updateNode(modificationNode);
     }
 
+    @Override
+    public void updateLoadFlowStatus(UUID nodeUuid, LoadFlowStatus loadFlowStatus) {
```

Comment:
So here you override impl of AbstractNodeRepositoryProxy ?
Why not removing the useless code in AbstractNodeRepositoryProxy ?
Would the best fix be to fix void updateLoadFlowStatus(AbstractNode node, LoadFlowStatus loadFlowStatus) instead to not replace the node when just updating the status?

Reply:
```java

    @Override
    public void updateLoadFlowStatus(AbstractNode node, LoadFlowStatus loadFlowStatus) {
        ModelNode modelNode = (ModelNode) node;
        modelNode.setLoadFlowStatus(loadFlowStatus);
        updateNode(modelNode);
    }
```

Reply:
Why do we update the node here?